strings/c-or-cpp/Permutation of String.c

```diff
@@ -0,0 +1,57 @@
+#include<stdio.h>
+//declare permutation and swap functions
+void permutation(char *,int,int);
+void swap(char *,char *);
+main()
+{
+    char s[10];
```

Comment:
what if a larger string is passed then 10 ?

Reply:
ok i will create the string length dynamically